changelogs/fragments/ini_file-preserve-symlink.yml

```diff
@@ -0,0 +1,3 @@
+---
+bugfixes:
+  - "ini_file - Follow the symlinks instead of replacing them (https://github.com/ansible-collections/community.general/issues/6470)."
```

Comment:
```suggestion
  - "ini_file - follow the symlinks instead of replacing them (https://github.com/ansible-collections/community.general/issues/6470, https://github.com/ansible-collections/community.general/pull/6546)."
```